torch/_torch_docs.py

```diff
@@ -1324,6 +1324,136 @@ def merge_dicts(*dicts):
     silently incorrect.
 """)
 
+add_docstr(torch.hsplit,
+           r"""
+hsplit(input, indices_or_sections) -> List of Tensors
+
+Split a tensor into multiple tensors horizontally (column wise).
+
+The input tensor is required to have at least one dimension. 
+
+This is equivalent to calling :func:`torch.tensor_split` with :attr:`dim=1` for 
```

Comment:
"This is equivalent to..." -> "If :attr:\`input\` is one dimensional this is equivalent to calling torch.tensor_split(input, indices_or_sections, dim=0) (the split dimension is zero), and if :attr:\`input\` has two or more dimensions it's equivalent to calling torch.tensor_split(input, indices_or_sections, dim=1) (the split dimension is 1), except that if :attr:\`indices_or_sections\` is an integer it must evenly divide the split dimension or a runtime error will be thrown."